cmd/alerts/silence/add_silence.go

```diff
@@ -26,22 +26,25 @@ type addSilenceCmd struct {
 func NewCmdAddSilence() *cobra.Command {
 	addSilenceCmd := &addSilenceCmd{}
 	cmd := &cobra.Command{
-		Use:               "add <cluster-id> [--all --duration --comment | --alertname --duration --comment]",
+		Use:               "add [--cluster-id <cluster-identifier>] [--all --duration --comment | --alertname --duration --comment]",
```

Comment:
If `--cluster-id` is required I think it should be not in brackets right?
```suggestion
		Use:               "add --cluster-id <cluster-identifier> [--all --duration --comment | --alertname --duration --comment]",
```